docs/fundamentals/networking/http/httpclient-guidelines.md

```diff
@@ -28,16 +26,19 @@ The preceding `HttpClient` is configured to reuse connections for 15 minutes. Af
 
 ## Pooled connections
 
-The connection pool for an <xref:System.Net.Http.HttpClient> is linked to its underlying <xref:System.Net.Http.HttpMessageHandler>. When the <xref:System.Net.Http.HttpClient> instance is disposed, it disposes all previously used connections. If you later send a request to the same server, a new connection is created. There's also a performance penalty because it needs a new TCP port. If the rate of requests is high, or if there are any firewall limitations, that can **exhaust the available sockets** because of default TCP cleanup timers.
+The connection pool for an <xref:System.Net.Http.HttpClient> is linked to the underlying <xref:System.Net.Http.SocketsHttpHandler>. When the <xref:System.Net.Http.HttpClient> instance is disposed, it disposes all existing connections inside the pool. If you later send a request to the same server, a new connection must be created. As a result, there's a performance penalty for unnecessary connection creation. Moreover, TCP ports are not released immediately after connection closure, see TCP `TIME-WAIT` in [RFC 9293](https://www.rfc-editor.org/rfc/rfc9293.html#section-3.3.2). So if the rate of requests is high, the operating system limit of available ports might be exhausted. To avoid port exhaustion problems, reusing <xref:System.Net.Http.HttpClient> instance for as many HTTP requests as possible is recommended. This can be achieved by following [the recommended usage](#recommended-use).
```

Comment:
```suggestion
The connection pool for an <xref:System.Net.Http.HttpClient> is linked to the underlying <xref:System.Net.Http.SocketsHttpHandler>. When the <xref:System.Net.Http.HttpClient> instance is disposed, it disposes all existing connections inside the pool. If you later send a request to the same server, a new connection must be recreated. As a result, there's a performance penalty for unnecessary connection creation. Moreover, TCP ports are not released immediately after connection closure, see TCP `TIME-WAIT` in [RFC 9293](https://www.rfc-editor.org/rfc/rfc9293.html#section-3.3.2). So if the rate of requests is high, the operating system limit of available ports might be exhausted. To avoid port exhaustion problems, reusing <xref:System.Net.Http.HttpClient> instance for as many HTTP requests as possible is recommended. This can be achieved by following [the recommended usage](#recommended-use).
```